tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -727,43 +736,39 @@ Function SkipIfPassive
   ${IfThen} $PassiveMode = 1  ${|} Abort ${|}
 FunctionEnd
 
-Function SkipIfPassiveButUpdateShortcutIfUpdate
-  ${If} $PassiveMode = 1
-    Call CreateDesktopShortcut
-    Abort
+Function CreateOrUpdateStartMenuShortcut
+  ; We used to use product name as MAINBINARYNAME
+  ; migrate old shortcuts to target the new MAINBINARYNAME
+  ${If} ${FileExists} "$DESKTOP\${PRODUCTNAME}.lnk"
```

Comment:
We should only update if in update mode, otherwise recreate the shortcut

```suggestion
  ; Update shortcuts target path if updating
  ${If} $UpdateMode = 1
  ${AndIf} ${FileExists} "$DESKTOP\${PRODUCTNAME}.lnk"
```

Reply:
We don't pass in /update in the earlier versions, and this migration is for those versions only, we shouldn't re-create in not /update in the command line args

Reply:
right, I forgot if we decided on a solution for this tbh, let's roll with this solution for now as it seems the safest